corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/sub_group.html

```diff
@@ -16,87 +16,92 @@
         class: elementTile,
         css: {
           'gr-no-children': $data.children().length === 0,
-          'gr-has-no-nested-questions': !$data.hasAnyNestedQuestions(),
-          'card': collapsible,
+          'd-none': !$data.hasAnyNestedQuestions(),
           'required-group': !showChildren() && childrenRequired(),
           'stripe-repeats': stripeRepeats,
-          'group-border': groupBorder,
+
         }">
-    <fieldset class="gr-header" data-bind="
+    <div data-bind="
```

Comment:
Add extra div for the border. Since I reverted back to using `row` and `col` for the spacing, the border would be outside the padding from the col and the group header inside. This causes a gap between the header background and the border. The new div is also inside the padding removing the gap.

Reply:
the rest is just indent here.

Reply:
feels weird to me to have a `div` just for borders, but i don't have a better suggestion.